### DISCUSSION TITLE
Support for Panasonic G9ii RW2 Format

### DISCUSSION THREAD

**Discussion by akjohnston** [ImageMagick/ImageMagick#7723] [General]
This seems to support older Panasonic RAW formats (such as for the S5 and G9) well, but is failing to correctly open images from newer cameras such as the G9ii. How do I go about checking what's formally supported, and request camera support if required?
Thanks
Andrew

**Comment by snibgo** [comment]
IM uses libraw to process raw camera files. https://www.libraw.org/supported-cameras shows the cameras currently supported by libraw. Your camera is not shown on that list. However, https://www.libraw.org/supported-cameras-202403 for the "202403 snapshot" does include "DC-G9 Mark II", which I suppose is your camera. I don't know how a "snapshot" relates to releases.

So I suppose your camera will be supported soon. Perhaps, if you download the snapshot and rebuild IM, that will include your camera.
